app/controllers/users/piv_cac_recommended_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Users
+  class PivCacRecommendedController < ApplicationController
+    include TwoFactorAuthenticatableMethods
+    include MfaSetupConcern
+    include SecureHeadersConcern
+    include ReauthenticationRequiredConcern
+
+    before_action :authenticate_user!
+    before_action :confirm_user_authenticated_for_2fa_setup
+    before_action :apply_secure_headers_override
+    before_action :user_email_is_gov_or_mil?
```

Comment:
1. This method doesn't return a boolean despite it suffixed with `?`
2. I almost wonder if we shouldn't bother with this. If someone wants to visit this page it's not a big deal? (We don't redirect away for second MFA reminder, for example)

Reply:
Yea I was wondering if we wanted to prevent users from seeing this if we dont want them to.

Reply:
I think it'd be fine to keep if we've already done the work, and could help avoid future issues if ever we added something or made assumptions about the users that requires them to have this email.

Only thing I'd recommend is calling it something to reflect what it's doing, e.g. `:confirm_user_email_is_gov_or_mil`, or `:redirect_unless_user_email_is_gov_or_mil`, or a conditional `before_action` like `before_action :redirect_to_after_sign_in_path, unless: :user_has_gov_or_mil_email?`